testing/xdg-desktop-portal-kde/APKBUILD

```diff
@@ -0,0 +1,34 @@
+# Contributor: Rasmus Thomsen <oss@cogitri.dev>
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
+pkgname=xdg-desktop-portal-kde
+pkgver=5.16.1
+pkgrel=0
+pkgdesc="Qt/KF5 implementation of xdg-desktop-portal"
+url="https://phabricator.kde.org/source/xdg-desktop-portal-kde"
+arch="all"
+license="GPL-3.0-or-later"
+makedepends="cmake qt5-qtbase-dev glib-dev kio-dev kwayland-dev libepoxy-dev
+	pipewire-dev kcoreaddons xdg-desktop-portal extra-cmake-modules cups-dev"
+subpackages="$pkgname-lang"
+source="https://download.kde.org/stable/plasma/$pkgver/xdg-desktop-portal-kde-$pkgver.tar.xz"
+
+build() {
+	if [ "$CBUILD" != "$CHOST" ]; then
+		CMAKE_CROSSOPTS="-DCMAKE_SYSTEM_NAME=Linux -DCMAKE_HOST_SYSTEM_NAME=Linux"
+	fi
+	cmake \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DCMAKE_INSTALL_LIBDIR=lib \
+		-DBUILD_SHARED_LIBS=True \
+		-DCMAKE_BUILD_TYPE=Release \
```

Comment:
This should probably have been built with RelWithDebInfo instead

Reply:
Huh, I just went with whatever newapkbuild gave me, maybe we should change it there then?